measure-cycle-time.html

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: "Time to allocate"
+bodyclass: ""
+---
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-xlarge">
+      <span class="heading-secondary">Guide</span>
+      How to measure cycle time
+    </h1>
+  </div>
+</div>
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+      <p>So how do we measure and improve Time to Allocate? First, some definitions.</p>
+      <h2 class="heading-small">Lead Time</h2>
+      <p>Lead Time is the period between a new task’s appearance in your workflow and its final departure from the system.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-diagram-1.png" class="limit-image" />
+      <h2 class="heading-small">Cycle Time</h2>
+      <p>Cycle Time begins at the moment when the new arrival enters “in progress” stage and somebody is actually working on it.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-diagram-2.png" class="limit-image" />
+      <h2 class="heading-small">Time to Allocate</h2>
+      <p>Time to Allocate is the time the task takes to reach the correct person to work on it.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-diagram-3.png" class="limit-image" />
+      <h2 class="heading-medium">What to measure?</h2>
+      <p>Most authorities track Lead Time against the statutory 20 day target, but just measuring Lead Time doesn’t provide enough resolution to focus efforts for improvements. All we know is whether we’re on target or not. Problems could be anywhere in the system.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-1.png" class="limit-image" />
+      <p>Lead time is comprised of Time to Allocate and Cycle Time. There’s more flexibility to target a specific area if we break up Lead Time in to its components.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-2.png" class="limit-image" />
+      <p>Each process is roughly independent so a delay in either area can contribute to a longer Lead Time.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-3.png" class="limit-image" />
+      <p>Cycle Time consists of many discrete tasks with predictable timescales, and a few unpredictable but critical tasks such as information retrieval.</p>
+      <p>Time to Allocate is “wasted” time as it's not directly contributing to the response. Reducing it provides more time for the “Cycle Time”, and could even build in some buffer time.</p>
+      <img src="{{site.baseurl}}/assets/img/lead-time-4.png" class="limit-image" />
```

Comment:
Worth using the updated images where the 20-day timeline is the same proportion https://docs.google.com/drawings/d/1DY0hzvCYM2GKLuhiw25efFMXI7mUPJ59rxKCzd91_Bc/edit?